### DISCUSSION TITLE
Access gists from mobile app

### DISCUSSION THREAD

**Discussion by llricci** [community/community#51566] [Mobile] [answered]
Is there a way to access gists from the mobile app? Or I have to open the browser and enter in my account from there? If so, I would suggest to add that as a future feature.

**Comment by iskae** [comment, marked as answer]
Hello @lucasricci 👋 

Thanks a lot for reaching out ❤️ 

Currently we are not supporting Gists in GitHub Mobile, but I will be forwarding your request and will be creating an internal issue for tracking purposes 👍 

Please stay tuned for future versions!

Cheers!

**Reply by llricci** [reply]
Thanks for the answer.
